src/app/make-cocktail/make-cocktail.component.scss

```diff
@@ -0,0 +1,196 @@
+.make-cocktail-page {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  font-family: "Architects Daughter", cursive;
+  overflow: hidden;
+  .ingredients-bar {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 100px;
+    width: 120px;
+    height: 100vh;
+    overflow-y: auto;
+    overflow-x: hidden;
+    color: #fff;
+    background-color: rgba(0, 0, 0, 0.5);
+    .ingredient {
+      width: 100px;
+      height: 120px;
+      text-align: center;
+      cursor: pointer;
+      img {
+        width: 100px;
+        height: 100px;
+      }
+      p {
+        margin: 0;
+      }
+    }
+  }
+  .cocktail-glass {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 150px 150px 50px 150px;
+    height: calc(100% - 200px);
+    width: calc(100% - 120px);
+    h1 {
+      position: absolute;
+      color: #fff;
+      top: 10%;
+    }
+    .glass-container {
+      width: 600px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: 600px;
+      img {
+        width: 600px;
+        height: auto;
+        position: absolute;
+      }
+    }
+    .custom-ingredients {
+      position: absolute;
+      top: 25%;
+      width: 400px;
+      z-index: -1;
+      img {
+        width: 75px;
+        height: 75px;
+        filter: blur(1px);
+      }
+    }
+    .shake {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-around;
+      width: 100%;
+      button {
+        margin: 10px 0;
+      }
+    }
+  }
+}
+
+@keyframes spaceboots {
```

Comment:
Better to use custom prefixes for custom animation for instance:
spaceboots --> ff-space-boots or eg-space-boots